### ISSUE TITLE
Barebones Script Issue

### ISSUE THREAD

**Issue by gregorylarsen** [happyleavesaoc/python-limitlessled#26]
Just wondering if I can get some feedback on a script I am running.
Script runs without error and ngrep verifies that I am definitely sending the commands to the bridge but no instructions are working.

Presently, my script is as follows:

from limitlessled.bridge import Bridge
from limitlessled.group.rgbww import RGBWW
bridge = Bridge('192.168.1.61')

bridge.add_group(1, 'living', RGBWW)
living = bridge.add_group(1, 'living', RGBWW)

living.on = True

**Comment by happyleavesaoc** [comment]
What version bridge are you using?

**Comment by gregorylarsen** [comment]
Bridge is v6.

Should that script be functional?

**Comment by happyleavesaoc** [comment]
Yes, it should work. Unfortunately I don't have a v6 bridge so I can't say for certain. You're adding the group twice, but that shouldn't break it.

**Comment by byxk** [comment]
Might be late to the party, but if you had setup a password for the bridge via the phone app, that would mess things up. Try restoring the bridge to factory settings and avoid setting the password.
